.github/patch-initial-checklist.md

```diff
@@ -59,7 +59,7 @@ Additionally, make sure to differentiate between things in the testing notes tha
   * If the base branch was `trunk`, and this release was deployed to WordPress.org, then merge the branch into `trunk`.
   * If the base branch is the release branch this patch release is for, then merge branch into release branch and then merge the release branch back to `trunk` if the patch release is the latest released version. Otherwise just merge back into the release branch.
 * [ ] If you merged the branch to `trunk`, then update version on the `trunk` branch to be for the next version of the plugin and include the `dev` suffix (e.g. something like [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158)) for the next version.
-* [ ] Update the schedules p2 with the shipped date for the release (Pca54o-1N-p2).
```

Comment:
It's good to use a tampermonkey script here as we ideally don't want internal urls to be public